dashboard/test/controllers/ai_tutor_interactions_controller_test.rb

```diff
@@ -0,0 +1,21 @@
+require 'test_helper'
+
+class AiTutorInteractionsControllerTest < ActionController::TestCase
+    setup do
+        @student_with_ai_tutor_access = create :student_with_ai_tutor_access
+    end
+
+    test "create AI Tutor Interaction with valid params" do
+        sign_in @student_with_ai_tutor_access 
+        assert_creates(AiTutorInteraction) do
+            post :create, params: {
+                level_id: 1234,
+                script_id: 987,
+                type: "general_chat",
+                prompt: "Can you help me?",
+                status: "ok",
+                ai_response: "Yes, I can help."
+            }
+        end
+    end
+end
```

Comment:
@kakiha11 Let's start with these other cases: 
- [ ] student without ai tutor access can NOT create an `AiTutorInteraction`
- [ ] can NOT create an `AiTutorInteraction` with an invalid `type` (ie. something other than "validation", "general_chat" or "compilation") 
- [ ] `AiTutorInteraction` is NOT created if the status is not "ok"
 
Then we can pair on a trickier case where `isProjectBacked == true` and we need more setup to get valid data.